<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenerateSinusoidImageDialogClass</class>
 <widget class="QDialog" name="GenerateSinusoidImageDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GenerateSinusoidImageDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Width</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="widthSpinBox">
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>400</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Height</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="heightSpinBox">
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>400</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>N</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="nSpinBox">
     <property name="maximum">
      <double>5000.000000000000000</double>
     </property>
     <property name="value">
      <double>256.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="sSpinBox">
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="generateButton">
     <property name="text">
      <string>Generate</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
